schema/spacewalk/common/tables/rhnPackage.sql

```diff
@@ -59,6 +59,14 @@ CREATE TABLE rhnPackage
     header_sig       VARCHAR(64),
     copyright        VARCHAR(128),
     cookie           VARCHAR(128),
+    is_ptf           NUMERIC(1)
+                         DEFAULT (0)
+                         CONSTRAINT rhn_package_is_ptf_check
+                             CHECK (is_ptf in ( 1 , 0 )),
+    is_part_of_ptf   NUMERIC(1)
+                         DEFAULT (0)
+                         CONSTRAINT rhn_package_is_part_of_ptf_check
+                             CHECK (is_part_of_ptf in ( 1 , 0 )),
```

Comment:
I wonder what is the reason to use `NUMERIC(1)` instead of [boolean]()?
Is it only because of "historic reason"?

There is quite size differences there:
```sql
# select pg_column_size(boolean 'TRUE'), pg_column_size(NUMERIC(1) '1');
pg_column_size | pg_column_size
----------------+----------------
             1 |              8
```

Reply:
This was just for historical reasons. I'll give it a try to use "boolean". Thanks for the note!

Reply:
Looking into some blogs wrt. postgres datatype performance, numeric type is one of the worst performant datatype. And I see we have plenty of them. 😄

Reply:
We also have plenty of `Y/N` char values :laughing:

Reply:
I had the same question as Ondrej. If we want to save a boolean flag we should use what database gives us for that :)

Reply:
This is now switched to use boolean in the DB